### PR TITLE
refactor!: remove CLI code

### PR DESCRIPTION
**Pull request by kaimallea**
BREAKING CHANGE: remove CLI code so that this package can focus on the
single responsibility of being a module.

### CODE REVIEW

**Comment by KenEucker** [comment]
@kaimallea 

I like the idea of removing the cli code and focusing on being a module. However, I was hoping we could retain the cli. Maybe we can create another project named imgur-cli and move the code to there? That way, if anyone is using the CLI right now they can continue to transition with the project. Also, if someone is motivated to invest in the CLI outside of the module I think keeping it around would benefit both projects.

File: README.md [hunk 1]
```diff
@@ -1,81 +1,3 @@
-## Command-line Usage
```
Comment (KenEucker) [inline README.md:1]: These lines could become the README for a new `imgur-cli` project.

File: cli.js [hunk 2]
```diff
@@ -1,175 +0,0 @@
-#!/usr/bin/env node
```
Comment (KenEucker) [inline cli.js:1]: This entire file could be moved to the new `imgur-cli` project.

**Review by KenEucker — COMMENTED** [review]
I suggest creating a new project named `imgur-cli` and I am willing to create and maintain that project separately. Using this project, I think it would be best to include the code in this project and display a deprecation warning for CLI features for a predetermined amount of time before completely removing CLI functionality from this package.

**Comment by kaimallea** [comment]
@KenEucker that was precisely my thinking, that it could be a separate package. I like your plan, I'll close this for now and we can properly deprecate it.

**Pull request closed without merging**